scripts/binary_search.sh

```diff
@@ -22,8 +22,8 @@ TESTOUTPUT=$WORKDIR/recaltable.cloud
 #
 # Set here the genomic interval you'd like to start from.
 #
-FIRST=1000250;
-LAST=1000312;
+FIRST=4000000;
+LAST=6000000;
```

Comment:
Recommend that you parameterize this script, as checking in updates to the various hardcoded values like this is not generally useful.